mobile-save-for-later/conf/cfn.yaml

```diff
@@ -34,11 +13,78 @@ Parameters:
     AllowedValues:
       - CODE
       - PROD
+
 Mappings:
   StageVariables:
     CODE:
-      TableReadCapacity: 1
-      TableWriteCapacity: 1
+      TableReadMinCapacity: 1
+      TableReadMaxCapacity: 10
+      TableWriteMinCapacity: 1
+      TableWriteMaxCapacity: 10
     PROD:
-      TableReadCapacity: 200
-      TableWriteCapacity: 125
+      TableReadMinCapacity: 50
+      TableReadMaxCapacity: 400
+      TableWriteMinCapacity: 50
+      TableWriteMaxCapacity: 400
+
+Resources:
+  SaveForLaterDynamoTable:
+    DeletionPolicy: Retain
+    Type: 'AWS::DynamoDB::Table'
+    Properties:
+      KeySchema:
+        - KeyType: HASH
+          AttributeName: userId
+      TableName: !Sub '${App}-${Stage}-articles'
+      AttributeDefinitions:
+        - AttributeName: userId
+          AttributeType: S
+      ProvisionedThroughput:
+        WriteCapacityUnits: !FindInMap [ StageVariables, !Ref Stage, TableWriteMinCapacity ] # Initial value, autoscaled with below resources
+        ReadCapacityUnits: !FindInMap [ StageVariables, !Ref Stage, TableReadMinCapacity ] # Initial value, autoscaled with below resources
+
+  SaveForLaterWritesScalableTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    Properties:
+      MaxCapacity: !FindInMap [StageVariables, !Ref Stage, TableWriteMaxCapacity]
+      MinCapacity: !FindInMap [StageVariables, !Ref Stage, TableWriteMinCapacity]
+      ResourceId: !Sub "table/${App}-${Stage}-articles"
+      RoleARN: !Sub arn:aws:iam::${AWS::AccountId}:role/aws-service-role/dynamodb.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_DynamoDBTable
+      ScalableDimension: dynamodb:table:WriteCapacityUnits
+      ServiceNamespace: dynamodb
+
+  SaveForLaterWritesScalingPolicy:
+    Type: AWS::ApplicationAutoScaling::ScalingPolicy
+    Properties:
+      PolicyName: SaveForLaterWritesScalingPolicy
+      PolicyType: TargetTrackingScaling
+      ScalingTargetId: !Ref SaveForLaterWritesScalableTarget
+      TargetTrackingScalingPolicyConfiguration:
+        TargetValue: 70.0
+        ScaleInCooldown: 60
+        ScaleOutCooldown: 10
+        PredefinedMetricSpecification:
```

Comment:
Nice. Just checked the docs to find out a bit more about the numbers here. Might be helpful to add a comment to say which units these are.

Reply:
Nice suggestion - the fact that we've both had to look up what units these are in makes a comment worthwhile here